### DISCUSSION TITLE
Whole year view

### DISCUSSION THREAD

**Discussion by varma02** [gpbl/react-day-picker#2649] [Support] [answered]
I would like for there to be an option to see all months and days on a single page without having to navigate between months.
Kinda like this but more pretty:
![image](https://github.com/user-attachments/assets/52ac4770-c064-4093-8ab9-890f53149124)

**Comment by gpbl** [comment, marked as answer]
Hi @varma02, to display 12 months set `numberOfMonths={12}`. 

```tsx
<DayPicker numberOfMonths={12} />
```

See https://daypicker.dev/docs/navigation and https://daypicker.dev/docs/customization#multiplemonths

![Screen Shot 2024-12-28 at 05 53 08](https://github.com/user-attachments/assets/5088c2ff-0a47-4d0c-8b14-33b0fe1d2adb)

**Reply by varma02** [reply]
Oh I didn't realize that was a thing 😅
Thank you!
